### ISSUE TITLE
Could you please update ember-es6_template dependency version?

### ISSUE THREAD

**Issue by dchuquilla** [emberjs/ember-rails#538]
I am getting this warning, I replaced ember-es6_template 0.4.4 to 0.5.3 vor trying to avoid is, but ember-rails only works with '0.4.-*'

```
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from setup at ~/.rvm/gems/ruby-2.4.0@tapinto/gems/ember-es6_template-0.4.4/lib/ember/es6_template/sprockets3.rb:10)
```

**Comment by tricknotes** [comment]
Sorry for my delay.
Now I published a new version of [ember-rails@0.21.0](https://rubygems.org/gems/ember-rails/versions/0.21.0) that uses the new version of ember-es6_template.

**Comment by dchuquilla** [comment]
Great thanks so much
